iot-hub/Quickstarts/read-d2c-messages/ReadDeviceToCloudMessages.js

```diff
@@ -1,55 +1,87 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-'use strict';
-
-// Connection string for the IoT Hub service
-//
-// NOTE:
-// For simplicity, this sample sets the connection string in code.
-// In a production environment, the recommended approach is to use
-// an environment variable to make it available to your application
-// or use an x509 certificate.
-// https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-security
-//
-// Using the Azure CLI:
-// az iot hub show-connection-string --hub-name {YourIoTHubName} --policy-name service  --output table
-var connectionString = '{Your service connection string here}';
-
-// Using the Node.js SDK for Azure Event hubs:
-//   https://github.com/Azure/azure-event-hubs-node
-// The sample connects to an IoT hub's Event Hubs-compatible endpoint
-// to read messages sent from a device.
-var { EventHubClient, EventPosition } = require('@azure/event-hubs');
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
+
+/*
+  This sample demonstrates how to use the Microsoft Azure Event Hubs Client for JavaScript to 
+  read messages sent from a device. Please see the documentation for @azure/event-hubs package
+  for more details at https://www.npmjs.com/package/@azure/event-hubs
+
+  For an example that uses checkpointing, follow up this sample with the sample in the 
+  eventhubs-checkpointstore-blob package on GitHub at the following link:
+
+  https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/eventhubs-checkpointstore-blob/samples/javascript/receiveEventsUsingCheckpointStore.js
+*/
+
+const { EventHubConsumerClient } = require("@azure/event-hubs");
+
+// If using websockets, uncomment the below require statement
```

Comment:
"the below require statement" -> "the require statement below"

Reply:
addressed in https://github.com/Azure-Samples/azure-iot-samples-node/pull/39/commits/058d62c74ce7afa6965a8dc09eced55af9e4a340